setup.py

```diff
@@ -30,9 +30,7 @@
     install_requires=[
         "appdirs",
         "bw_processing",
-        "fasteners",
         "peewee>=3.9.4",
-        "wrapt",
```

Comment:
We can't remove `wrapt` we are using it for [tests](https://github.com/baali/bw_projects/blob/main/tests/projects.py) and creating `tempdir` during the tests.

Reply:
I am [shifting](https://github.com/baali/bw_projects/pull/13/commits/358d1ceaa1904e1e1fba88fca45f5df31a61a4db) to other approach of handling `tempdir` and cleaning them up.